src/privileged/FirefoxMonitor.css

```diff
@@ -0,0 +1,85 @@
+#fxmonitor-notification .popup-notification-body {
+  font-size: 110%;
+}
+
+#fxmonitor-notification popupnotificationcontent {
+  margin-top: 0;
+}
+
+#fxmonitor-notification .popup-notification-body > hbox:first-child {
+  display: none;
+}
+
+#fxmonitor-notification .popup-notification-icon,
+.fxmonitor-icon {
+  list-style-image: url("chrome://browser/skin/warning.svg");
```

Comment:
I'd generally advise to ship your own warning.svg, but I would be okay with this, just don't complain when someone breaks/replaces it, since they can't searchfox your usage.